gradle.properties

```diff
@@ -1,3 +1,3 @@
-grailsVersion=3.3.0.RC1
-gormVersion=6.1.5.RELEASE
-gradleWrapperVersion=4.0
+grailsVersion=3.3.1
+gormVersion=6.1.7.RELEASE
+gradleWrapperVersion=3.5
```

Comment:
Unfortunately I need Gradle version >= 4.0 because the version of gradle-vaadin-plugin that works reliably with Spring Boot apps (and hence, with Grails) doesn't work with Grails 3.x